Gopkg.toml

```diff
@@ -54,6 +54,10 @@
     name = "k8s.io/client-go"
     branch = "release-8.0"
 
+[[override]]
+    name = "gopkg.in/DataDog/dd-trace-go.v1"
+    version = "=v1.5.0"
```

Comment:
https://github.com/DataDog/dd-trace-go/releases/tag/v1.6.0. was just released with the support of `DD_AGENT_HOST` & `DD_TRACE_AGENT_PORT`